test/controllers/BaseControllerSpec.scala

```diff
@@ -37,7 +39,7 @@ import scala.concurrent.Future
 /**
   * Each Controller test needs to extend this class to simplify controller testing
   */
-abstract class BaseControllerSpec extends BaseSpec with ScalaFutures {
+abstract class BaseControllerSpec extends UnitWithAppSpec with ScalaFutures with MockitoSugar {
```

Comment:
UnitWithAppSpec extends UnitSpec which has mixins ScalaFutures and MockitoSugar already